### PR TITLE
[SYCL] Add test for queue.get_property<in_order>

### PR DESCRIPTION
**Pull request by denis-kabanov**

### CODE REVIEW

File: SYCL/InorderQueue/in_order_get_property.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+
+int main() {
+  queue Queue1;
+  try {
+    Queue1.get_property<property::queue::in_order>();
+    assert(false && "Queue1 was created without any propertys therefore get "
```
Comment (s-kanaev) [inline SYCL/InorderQueue/in_order_get_property.cpp]:
```suggestion
    assert(false && "Queue1 was created without any properties therefore get "
```

**Review by s-kanaev — DISMISSED** [review]
LGTM

File: SYCL/InorderQueue/in_order_get_property.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+
+int main() {
+  queue Queue1;
+  try {
+    Queue1.get_property<property::queue::in_order>();
```
Comment (s-kanaev) [inline SYCL/InorderQueue/in_order_get_property.cpp:25]: Should we check for has_property here too?

Comment (denis-kabanov) [reply SYCL/InorderQueue/in_order_get_property.cpp:25]: has_property check added.

**Review by s-kanaev — APPROVED** [review]

**Pull request merged**